builders/toolkit/.pages

```diff
@@ -5,3 +5,4 @@ nav:
   - ethereum-api
   - substrate-api
   - integrations
+  - rpc.md
```

Comment:
Since this article focuses only on Ethereum RPC calls, shouldn't it be placed in the ethereum-api section?  wdyt @eshaben @themacexpert ?

Reply:
Yeah, we do have the JSON-RPC page on Moonbeam under the Ethereum section. Works for me. Good catch @augustocollado